docs/conf.py

```diff
@@ -15,7 +15,13 @@
 
 sys.path[:0] = [
```

Comment:
probably too much work but this feels like it could be converted to a script that just looks for "src" directories under ../ext/

Reply:
Good idea, this will reduce the amount of changes someone needs to make when adding an integration. Submitted a change.